Update home tab

vtushar06 opened the pull request:
<!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
<!-- Ideally, a GitHub Issue with the label `Status: Consensus` will have been concluded, and linked to in this PR. -->
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**

- Feature and content update

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**
<!-- Pick one of the below options.  Please remove those which don't apply. -->
-  Closes #69 


**Screenshots/videos:**
<img width="1512" height="982" alt="Screenshot 2025-09-13 at 2 27 46 PM" src="https://github.com/user-attachments/assets/ec3a6a5f-43a6-44dd-9bdb-a60d127c8a84" />
<img width="1512" height="982" alt="Screenshot 2025-09-18 at 2 37 09 PM" src="https://github.com/user-attachments/assets/7372586e-d78f-420a-9c2f-fcfe3368fcb4" />
<img width="1512" height="982" alt="Screenshot 2025-09-18 at 2 36 34 PM" src="https://github.com/user-attachments/assets/3f0c4912-6124-4d74-88a1-919cf326ef86" />


<!--Add screenshots or videos wherever possible.-->

**If relevant, did you update the documentation?**
No documentation update required

<!--Add link to it-->

**Summary**

<!-- Explain the motivation for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->
This PR implements the updates requested in issue #69 to the Home tab of the JSON Schema Conference website for 2025. The key changes include:

- Updated landing image using a photo from last year’s conference with overlay for clarity.
- Event date updated to “December 11th, 2025.”
- Added CTAs (“Apply to Speak”, “Become a Sponsor”, “Meet Speakers”, “Get Tickets”) aligned and placed directly below the event date.
- Added a countdown timer component under the CTAs.
- Redesigned the informational section to improve visual alignment and clarity.
- Removed the featured speakers section as per request.
- Retained the existing map section using Free API support.
- Added Past Sessions section displaying the first two recorded session videos from the official YouTube channel.
- Added a Tickets section with a visual design and CTA linking to the official ticketing platform.
- Updated the Sponsors section to include a “Become a Sponsor” CTA linking to the current sponsorship page and past sponsors including Octue, organized by category.
- Ensured visual style consistency with the conference branding guidelines.

**Does this PR introduce a breaking change?**
No

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

vtushar06 committed:
added background for paris

vtushar06 committed:
added countdown timer

vtushar06 committed:
updated map section and removed featured speakers

vtushar06 committed:
added past sessions section

vtushar06 committed:
updated past-sessions

vtushar06 committed:
added ticket section

vtushar06 committed:
updated tickets section

vtushar06 committed:
added ticket-cta

vtushar06 committed:
added sponsers in home page and logo from landscape

vtushar06 committed:
updated sponsers section for platinum and bronze and improved styling for pastsessions

Honyii commented:
@vtushar06 thanks for doing this work and creating a pr on time. However, some checklist were not met.

- The landing page image wasn't from the pictures taken from last year's comment. kindly check the main issue and navigate the steps to get these.
- In the countdown timer component, remove "conference starts in:" leave the rest as is.
- Take the countdown component up and bring the cta's down (the position where the countdown component currently is).
- The "what is json schema ...." section was not changed at all. If there's no design idea for this, then that's fine. 
- The past sessions should come in the youtube form with the thumbnail. We need to see that it is viewable on youtube once we scroll here (basically should be displayed as it is on youtube).
- The tickets section was not done as stated in the original issue, kindly check the description in the issue checklist (it should look like a card or modal).
- For sponsors, octue was a 2024 sponsor and should reflect that. So it can show become a sponsors cta, then paast sponsors (as octue, a platinum sponsor).
- Also for now, i am only sure of Octue, i referenced the landcape project so you could see here to take the octue logo/ information from. Also only the logo is needed for octue, kindly remove the other sponsors.

Altogether , thank you for your hard work.

vtushar06 commented:
Hii @Honyii, I am thinking to implement tickets section like this, is this looks clean and okay to you or need some changes 
I think for background color for Cta should match website theme color, that's why I used same.
<img width="1512" height="982" alt="Screenshot 2025-09-16 at 7 52 16 AM" src="https://github.com/user-attachments/assets/f51a2118-7943-4e65-89fe-f0485c05cc60" />

vtushar06 committed:
upgradition of landing page and styles

Honyii commented:
@vtushar06 this looks good, however, the designs for button should be the same across the website. Really any of it works but should be uniform. So @jagpreetrah kindly sync with Tushar so you can decide on what button style to use across the pages.
Also check the existing design for sessions and schedule - so you could update the current design for tickets. Alltogether, looks good!
cc @animeshsahoo1 so you could see what was agreed on pending when you start working on your issue.

vtushar06 commented:
yeah sure @Honyii, happy that you liked this, will sync with @jagpreetrahi for further updation. thanks and also Is there any kind of deadline for this issue as I think we should do this as fast as possible.

vtushar06 committed:
added padding for ticket container

vtushar06 commented:
hey @Honyii, I made some changes regarding styles and layout for home tab and I did not make changes for "what is json schema ...." section, as i think for now it is good. Thanks can you review this out whenever you are free so that we can move forward with changes. Thanks😊

Honyii commented:
@vtushar06 I've gone through what you've done, welldone. But can you take your time to go over the checklists one step at a time. Let's start from the things that were missed from the start;

- The tickets should route to the tickets page. Check the original issue, you will see where it should route to.
- For sponsorship, Octue was a 2024 sponsor as stated in the first reply here. So reflect that and only the logo is needed no need to add short description (as previously stated).

Then for new observation;

- JSON Schema name and logo is too at the top of the page, there is no spacing currently.  See how it is right now and do same.
<img width="1431" height="389" alt="Screenshot 2025-09-18 at 12 32 01" src="https://github.com/user-attachments/assets/2f039cc0-7ea8-4564-94d9-7b1c5552e635" />

- The button is too big, kindly reduce. Use the same size as it is currently (in the previous featured speakers section on the website).
<img width="1210" height="107" alt="Screenshot 2025-09-18 at 12 32 10" src="https://github.com/user-attachments/assets/033f0819-7ab4-4eb3-b1cc-1f40cb0777e0" />
Note: All the buttons in this page is too big - both in "get your tickets" and "become a sponsor".

- This one is new due to the changes we have, now the "dangling icon" looks off. Kindly take it below the conference date. I.e at the top of "secure your spot ....) modal.

Important: The new implementation is not well mobile responsive. Kindly make the buttons and all fit well on mobile.

New req: Kindly make the map section to have blank spaces on both sides, just as you have the "past sessions" Because right now, as the map works when scrolling, it disrupts the experience. 
Kindly leave some spaces on right and left side (but not too much as in the past session section).

vtushar06 committed:
updated maps and cta sizing

vtushar06 commented:
Hey @Honyii, I have made changes related to your's last comment 
Ensuring following things:
1. tickets cta should route to the mentioned link
2. reduced sizes for cta's 
3. provided spacing from top for JSON Schema logo
4. moved scroller animation above countdown
5. adjusted map as mentioned
6. removed description from octue logo
7. ensured mobile responsive changes

And if any more changes required or some suggestion for these changes love to hear.
Thanks

Honyii commented:
Thank you @vtushar06 this looks really good and I'd merge this for now so we can have this out and do cleanup in a final ticket for all sessions.
Currently, the links were supposed to route to a new page instead of in-page. Also the mobile responsiveness for the first modal and buttons are not well aligned but I do't think it's a blocker.
<img width="585" height="844" alt="Screenshot 2025-09-19 at 15 22 28" src="https://github.com/user-attachments/assets/a585513d-b8e7-4526-94ec-9399daf4b591" />

animeshsahoo1 commented:
@vtushar06  i cant see the timer working on the deployed version but i can see it work in development the issue might be related to time methods of javascript please take a look at it, if you are busy let me know i will look into it

<img width="1920" height="1030" alt="Image" src="https://github.com/user-attachments/assets/f4e7b8fc-a3ef-439c-aea2-34bbf9811d00" />

vtushar06 commented:
yes @animeshsahoo1 , we have discussed that and will fix this in final ticket. thanks for reminding us.
